gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/BridgeServerAdvisor.java

```diff
@@ -62,7 +63,24 @@ protected Profile instantiateProfile(
       InternalDistributedMember memberId, int version) {
     return new BridgeServerProfile(memberId, version);
   }
-  
+
+  @Override
+  public Set adviseProfileRemove() {
+    Set results = super.adviseProfileRemove();
+    // Add other members as recipients which are neither locators nor running netservers,
+    // if this is a snappy node. This could be helpful if a Spark Driver process is
+    // connected as a peer to this DS in split-cluster mode. SNAP-737
+    if (GemFireCacheImpl.getInternalProductCallbacks().isSnappyStore()) {
+      Set others = getDistributionManager().getOtherNormalDistributionManagerIds();
+      for (Object m : others) {
+        if (!results.contains(m)) {
```

Comment:
Since we are iterating over a set, I don't think contains check is required if results will not contain 'normalDistributionManagerIds' from 'super.adviseProfileRemove();' but yes no harm in checking.